doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -132,6 +132,13 @@ Matter
   * Page about :ref:`ug_matter_device_optimizing_memory`.
   * Shell commands for printing and resetting the peak usage of critical system resources used by Matter.
     These shell commands are available when both :kconfig:option:`CHIP_LIB_SHELL` and :kconfig:option:`CHIP_STATISTICS` Kconfig options are set.
+  * Reaction to removing the last fabric.
+    The user now decides what happens after the removal:
+    
+    * Do nothing (:kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_NONE`).
+    * Perform the factory reset of the device (:kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_ONLY`).
+    * Perform the factory reset of the device and start Bluetooth LE advertising (:kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START`).
```

Comment:
```suggestion
    * Perform a factory reset of the device and start Bluetooth LE advertising (:kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START`).
```
I'm not seeing the trailing whitespace that GitHub is reporting, but check that it's not there.